### ISSUE TITLE
Cannot Create Project Due To Uncaught Invalid or unsupported zip format. No END header found

### ISSUE THREAD

**Issue by teclsx** [joeferraro/MavensMate-Atom#121]
Attempt to create new project, but keep on throwing below error. Not sure what happen.

Atom Version: 0.165.0
MavensMate-Atom, v0.0.14
System: Microsoft Windows 7 Enterprise 

### Stack Trace
Uncaught Invalid or unsupported zip format. No END header found
At ..\.atom\packages\MavensMate-Atom\node_modules\mavensmate\node_modules\adm-zip\zipFile.js:66

**Comment by briansieb** [comment]
I just ran into the same error on Mac Yosemite when selecting the AuraDefinitionBundle metadata type:

[Enter steps to reproduce below:]

1. Select MavensMate > Project > Edit Project
2. Under Project Metadata tab select AuraDefinitionBundle
3. Click Update Project

**Atom Version**: 0.187.0
**System**: Mac OS X 10.10.2
**Thrown From**: [MavensMate-Atom](https://github.com/joeferraro/MavensMate-Atom) package, v0.0.20

### Stack Trace

Uncaught Invalid or unsupported zip format. No END header found

```
At /Users/bsiebert/.atom/packages/MavensMate-Atom/node_modules/mavensmate/node_modules/adm-zip/zipFile.js:66

undefined
```

### Commands

```
     -8:30.6 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-flatland-dark-ui)
     -2:40.7 command-palette:toggle (atom-text-editor.editor)
     -2:31.9 core:confirm (atom-text-editor.editor.mini)
     -2:31.9 mavensmate:open-project (atom-text-editor.editor)
 13x -0:42.6 core:move-down (atom-text-editor.editor.mini)
     -0:39.4 core:move-up (atom-text-editor.editor.mini)
     -0:38.7 core:confirm (atom-text-editor.editor.mini)
     -0:31.3 core:select-up (atom-text-editor.editor)
     -0:23.5 command-palette:toggle (atom-text-editor.editor)
     -0:18.9 core:confirm (atom-text-editor.editor.mini)
     -0:18.9 mavensmate:edit-project (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "flatland-dark-ui",
      "atom-dark-syntax"
    ]
  },
  "MavensMate-Atom": {
    "mm_workspace": [
      "/Users/bsiebert/Documents/mavensmate"
    ],
    "mm_panel_height": 336
  }
}
```

### Installed Packages

```coffee
# User
MavensMate-Atom, v0.0.20
flatland-dark-ui, v0.2.3
tabs-to-spaces, v0.8.1

# Dev
No dev packages
```

**Comment by tstachl** [comment]
On Mac this only happens with `AuraDefinitionBundle` resources.

**Comment by joeferraro** [comment]
this should be resolved in v1.0.0. please feel free to re-open if you're still experiencing this issue. thanks!
